docs/sphinx/using/examples/multi_gpu_workflows.rst

```diff
@@ -53,74 +39,74 @@ NVIDIA GPUs vs CPUs:
 
     { 0000000000000000000000000:510 1111111111111111111111111:490 }
 
-If one incrementally increases the qubit count, we
-reach a limit where the memory required is beyond the capabilities of a
-single GPU: A :math:`n` qubit quantum state has :math:`2^n` complex amplitudes, each
-of which require 8 bytes of memory to store. Hence the total memory
-required to store a :math:`n` qubit quantum state is :math:`8` bytes
-:math:`\times 2^n`. For :math:`n = 30` qubits, this is roughly :math:`8`
-GB but for :math:`n = 40`, this exponentially increases to 8700 GB.
 
-
-Parallelization across Multiple Processors
+Pooling the memory of multiple GPUs (`mgpu`)
 ---------------------------------------------
 
-The ``nvidia-mgpu`` target allows for memory from additional
-GPUs to be pooled enabling qubit counts to be scaled.
-Execution on the ``nvidia-mgpu`` backend is enabled via ``mpirun``. Users
-need to create a ``.py`` file with their code and run the command below
-in terminal:
 
-``mpirun -np 4 python3 test.py``
+As :code:`N` gets larger, the size of the state vector that needs to be stored in memory increases exponentially. 
+The state vector has :math:`2^N` elements, each a complex number requiring 8 bytes. This means a 30 qubit simulation 
+requires roughly 8 GB. Adding a few more qubits will quickly exceed the memory of as single GPU.  The `mqpu` backend 
+solved this problem by pooling the memory of multiple GPUs across multiple nodes to perform a single state vector simulation. 
+
+
+.. image:: images/mgpu.png
+
+
+If you have multiple GPUs, you can use the following command to run the simulation across :math:`n` GPUs. 
+
+
+:code:`mpiexec -np n python3 program.py --target nvidia --target-option mgpu`
+
+This code will execute in an MPI context and provide additional memory to simulation much larger state vectors.
+
 
-where 4 is the number of GPUs one has access to and ``test`` is the file
-name chosen.
+Batching Hamiltonian Terms (`mqpu`)
```

Comment:
I think that batching hamiltonian term is an example of mqpu. Let's call it something like 'Parallel execution over multiple QPUs'